client/src/components/Search.jsx

```diff
@@ -34,9 +35,9 @@ const buildSearchUrl = (coopSearchSettings, setSearchUrl) => {
       "name=" + encodeURIComponent(coopSearchSettings.name)
     );
   }
-  if ("type" in coopSearchSettings && coopSearchSettings.type != "") {
+  if ("type" in coopSearchSettings && coopSearchSettings.type != []) {
     individualSearchSettings.push(
-      "coop_type=" + encodeURIComponent(coopSearchSettings.type)
+      "coop_type=" + encodeURIComponent(JSON.stringify(coopSearchSettings.type))
```

Comment:
So the backend still needs a little bit of work (which I'm working on), but could you make a small formatting change here?  Per my running a test in the console, looks like JSON.stringify will turn an array into a string like so

```
types = ['abc', 'def']
(2) ['abc', 'def']
JSON.stringify(types)
'["abc","def"]'
```

Can we make it so the query part looks like "coop_type=museum,garden,credit+union" (as an example)?

Reply:
yah no problem